packages/did/package-lock.json

```diff
@@ -415,12 +398,11 @@
       "requires": {
         "@iota/converter": "^1.0.0-beta.11",
         "@iota/core": "^1.0.0-beta.11",
-        "@iota/curl": "^1.0.0-beta.11",
-        "babel-polyfill": "^6.26.0"
+        "@iota/curl": "^1.0.0-beta.11"
       }
     },
     "mam.tools.js": {
-      "version": "git+https://github.com/TangleID/mam.tools.js.git#3553c7ad79da450d7a7149e0cf882e36b8a7424c",
+      "version": "git+https://github.com/TangleID/mam.tools.js.git#04dedce992cb96f33649b8968e6f2ddb2eb659b4",
```

Comment:
Can we specify the tag or branch name instead?

Reply:
No, I cannot do that, because the lock file is automatically generated by the NPM.
It tracks the git hash to ensure everyone using the same version of the packages.

Reply:
I dislike the way to track `package-lock.json` in Git repository. However, I would merge this pull request first.

Reply:
I don't like it either, it will be removed in the future.